CHANGELOG.md

```diff
@@ -24,6 +24,17 @@
 - iOS: Sync metadata can be reset using the `resetBookmarksMetadata` method
   ([#1092](https://github.com/mozilla/application-services/pull/1092))
 
+## Push
```

Comment:
If this lands after #1119 (which seems very likely!) this should be in PENDING.md

Reply:
Heh, I was debating creating that myself since I didn't see it in the repo. I'll keep an eye out for it once I do a last pull from master.

Reply:
it's now CHANGES_UNRELEASED.md after some bikeshedding (this at least is next to CHANGELOG in the file list)